installer/mod.ts

```diff
@@ -86,69 +87,104 @@ function createDirIfNotExists(path: string): void {
   }
 }
 
-function checkIfExistsInPath(path: string): boolean {
-  const { PATH } = env();
+function checkIfExistsInPath(filePath: string): boolean {
+  // In Windows's Powershell. $PATH not exist. use $Path instead.
+  // $HOMEDRIVE is only use in Windows.
+  const { PATH, Path, HOMEDRIVE } = env();
```

Comment:
// In Windows's Powershell $PATH not exist, so use $Path instead.
// $HOMEDRIVE is only used on Windows.